### DISCUSSION TITLE
Cannot get programmatically updating a currency input to work

### DISCUSSION THREAD

**Discussion by frankmichel** [dm4t2/vue-currency-input#279] [Q&A] [answered]
I am trying to update a second readonly currency input based on another currency input. However, I just cannot get it to work in the most recent version. The reason why I am using the currency-input component on the readonly field is that I want to make use of the formatting options by simply providing a calculated number.

I have created a code sandbox to illustrate the problem:

https://codesandbox.io/s/green-cache-6oidb

I have been using the code snippet as explained in https://dm4t2.github.io/vue-currency-input/guide.html#external-props-changes, but still it does not work. Any suggestions or tips?

**Comment by dm4t2** [comment, marked as answer]
Hi, you need to remove the `value` assignment in the component when not using `v-model`:

https://codesandbox.io/s/lucid-moon-dgcio?file=/src/components/CurrencyInput.vue

It's actually not required for simple HTML input elements.

**Reply by frankmichel** [reply]
Thank you so much!! This is it :)
